### PR TITLE
Add ability to add external links from the site.xml

### PR DESCRIPTION
**Pull request by seanfeld**
# Add external Links to the cog menu on the main app bar

## Description
Changed server to allow for external links for the app menu bar as long as you use the correct property in the site.xml:
```
    <property>
    <name>uiExternalLinks.neato</name>
    <value>burrito.com</value>
    <description>
      Cool external link!
    </description>
  </property>
  ```

The script `prestart` is deprecated in newer versions of yarn so changed yarn start to reflect that.
Fixed eslint errors inside express.js

## PR Type
- [ ] Bug Fix
- [x] Feature
- [ ] Build Fix
- [ ] Testing
- [ ] General Improvement
- [ ] Cherry Pick

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -93,23 +94,46 @@ async function setAllowedOrigin() {
   }
 }
 
-log.info('Starting CDAP UI ...');
+log.info('Starting CDAP UI ...', 'goose');
```
Comment (mrahanjam) [inline server.js]: goose?

Comment (seanfeld) [reply server.js]: ahh i left a goose in there

Comment (seanfeld) [reply server.js]: I normally log `goose` places because its extremely unlikely that word is used elsewhere

File: server/express.js [hunk 2]
```diff
@@ -640,7 +649,7 @@ function makeApp(authAddress, cdapConfig, uiSettings) {
         var date = new Date();
         date.setDate(date.getDate() + 365); // Expires after a year.
         if (!req.cookies.bcookie) {
-          res.cookie('bcookie', uuidV4(), { ...cookieSettings, expires: date } );
+          res.cookie('bcookie', uuidV4(), { ...cookieSettings, expires: date });
```
Comment (njbriggs) [inline server/express.js]: This goes away

**Review by njbriggs — APPROVED** [review]

File: server/express.js (hunk 2 above)
Comment (seanfeld) [reply server/express.js]: ahh crap the rebase got screwed up

**Review by sumengwang — APPROVED** [review]

**Review by mrahanjam — APPROVED** [review]

**Pull request merged**